css/src/tokens/aspect-ratios.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * @sass-export-section="aspect-ratios"
+ */
+$aspect-ratios: (
+	'square': '1 / 1',
```

Comment:
shall we keep 1-1 so that our atomic names continue to follow their conventions?

Reply:
updated

Reply:
Yeah, we don't need to follow the older ways, more important to keep things consistent with the new conventions.